### PR TITLE
buildlib: fix build-sync trigger

### PR DESCRIPTION
**Pull request by sosiouxme**
(and a few other things)

### CODE REVIEW

**Comment by sosiouxme** [comment]
@tbielawa

File: pipeline-scripts/buildlib.groovy [hunk 1]
```diff
@@ -714,49 +715,45 @@ def build_ami(major, minor, version, release, yum_base_url, ansible_branch, mail
...
     }
     def fullVersion = "${major}.${minor}"
     try {
-        build(job: 'build%252Fbuild-sync', parameters:
-	      [
-		param('Choice', 'BUILD_VERSION', fullVersion)
-	    ]
-	)
+        build(job: 'build%2Fbuild-sync', parameters:
+            [ param('String', 'BUILD_VERSION', fullVersion) ]  // https://stackoverflow.com/a/53735041
```
Comment (tbielawa) [inline pipeline-scripts/buildlib.groovy]: You are the best

File: pipeline-scripts/buildlib.groovy [hunk 2]
```diff
@@ -714,49 +715,45 @@ def build_ami(major, minor, version, release, yum_base_url, ansible_branch, mail
...
             to: "${mail_list}",
             from: "aos-team-art@redhat.com",
             subject: "Error syncing images after ${fullVersion} build #${build_number}",
             body: [
                 "Encountered an error: ${err}",
-                "Input URL: ${env.BUILD_URL}input",
-                "Jenkins job: ${env.BUILD_URL}"].join('\n')
-            )
-        // Continue on, this is not considered a fatal error
+                "Jenkins job: ${env.BUILD_URL}"
```
Comment (tbielawa) [inline pipeline-scripts/buildlib.groovy]: nice touch

File: jobs/build/ocp4/Jenkinsfile [hunk 3]
```diff
@@ -809,14 +809,14 @@ Jenkins job: ${env.BUILD_URL}
...
-		BUILD_VERSION_MAJOR,
-		BUILD_VERSION_MINOR,
-		"aos-team-art@redhat.com",
-		currentBuild.number
-	    )
-	}
+        stage('sync images') {
+            buildlib.sync_images(
+                BUILD_VERSION_MAJOR,
+                BUILD_VERSION_MINOR,
```
Comment (tbielawa) [inline jobs/build/ocp4/Jenkinsfile]: I gotta figure out why `$EDITOR` is having such a time with tabs and space.

**Pull request merged**